statistics/handle/update.go

```diff
@@ -575,9 +576,10 @@ func (h *Handle) UpdateStatsByLocalFeedback(is infoschema.InfoSchema) {
 				}
 				newIdx := *idx
 				eqFB, ranFB := statistics.SplitFeedbackByQueryType(fb.Feedback)
-				// For StatsVersion higher than Version1, the topn is extracted out of histogram. So we don't update the histogram if the feedback overlaps with some topn.
 				if idx.StatsVer >= statistics.Version2 {
-					ranFB = statistics.CleanRangeFeedbackByTopN(ranFB, idx.TopN)
+					// // For StatsVersion higher than Version1, the topn is extracted out of histogram. So we don't update the histogram if the feedback overlaps with some topn.
+					// ranFB = statistics.CleanRangeFeedbackByTopN(ranFB, idx.TopN)
```

Comment:
Should these comments be updated?

Reply:
Originally, this is for using feedback in version2 stats.
We have decided that version2 stats is incompatible with feedback so they're useless now. But I still kept these codes for reference maybe in the future.
If needed, I can also delete this directly or add another comment to clarify this.
How do you think of it?